CMakeLists.txt

```diff
@@ -92,8 +92,7 @@ if(CMAKE_C_COMPILER_ID MATCHES "Clang")
   set_source_files_properties(src/mem/mem.c PROPERTIES COMPILE_FLAGS -Wpadded)
 endif()
 
-set(re_DIR ${CMAKE_CURRENT_LIST_DIR}/cmake)
-find_package(re CONFIG REQUIRED)
+include("${CMAKE_CURRENT_LIST_DIR}/cmake/re-config.cmake")
```

Comment:
I guess this change uncovered that the following `HINTS` don't work:
https://github.com/baresip/re/blob/bcf197e0e6d88e758dd3d342961b22e775b79992/test/CMakeLists.txt#L62